react-common/components/util.tsx

```diff
@@ -91,4 +91,25 @@ export function screenToSVGCoord(ref: SVGSVGElement, coord: ClientCoordinates) {
     screenCoord.x = coord.clientX;
     screenCoord.y = coord.clientY;
     return screenCoord.matrixTransform(ref.getScreenCTM().inverse());
+}
+
+export function findNextFocusableElement(elements: HTMLElement[], focusedIndex: number, index: number, forward: boolean): HTMLElement {
```

Comment:
nit: you can eliminate the focusedIndex argument by using the iterative implementation above instead of the recursive one

Reply:
I prefer using recursion for this, but would like to hear more of your thoughts if you feel strongly about implementing it with iteration.

Reply:
the iterative implementation doesn't require passing in focusedIndex. I just also generally prefer loops to tail recursion, personally (but feel free to ignore)